tests/ha/ha_cluster_init.pm

```diff
@@ -47,8 +57,8 @@ sub run {
     # Execute csync2 to synchronise the configuration files
     exec_csync;
 
-    # State of SBD
-    assert_script_run "sbd -d $sbd_device list";
+    # State of SBD if shared storage SBD is used
+    assert_script_run "sbd -d $sbd_device list" if !(get_var('USE_DISKLESS_SBD'));
```

Comment:
You could write this also as `assert_script_run "sbd -d $sbd_device list" unless get_var('USE_DISKLESS_SBD');`, whichever you think it's easier to read.

Reply:
`unless` is definitively easier to read.